### PR TITLE
VBF diboson cards

### PR DESCRIPTION
**Pull request by IreneZoi**
added cards for VBF diboson channels (except Spin-0/Radion_VBF_HH_inclu)

### CODE REVIEW

**Comment by perrozzi** [comment]
Hi, we strongly suggest to provide only example datacards for 1 mass point and a script to produce the others, otherwise the review of the PR is ~impossible and the genproductions repository gets overloaded

**Comment by IreneZoi** [comment]
Hi, ok, I'm doing the modification. Should I then do a new pull request?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_customizecards.dat~ [hunk 1]
```diff
@@ -0,0 +1,3 @@
+set param_card mass 35 800
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_customizecards.dat~]: please remove .dat~ files

Comment (IreneZoi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_customizecards.dat~]: yes I realized it too late, I am working on it, sorry

**Comment by IreneZoi** [comment]
Hi, sorry, any news about this pull request?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 0        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat]: I believe pdfwgt should be set to True for 2017 samples

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+ 0.4  =  dparameter 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 4 = maxjetflavor    ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+ 0   = xqcut   ! minimum kt jet measure between partons
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat]: xqcut = 0 is usually a bit too inclusive and can lead to mg5_amc failures. which qcut is used in pythia? can this be increased a bit?

Comment (IreneZoi) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat]: xqcut should be used only for MLM matching that is not done here so it shouldn't affect

**Comment by efeyazgan** [comment]
The cards look good to me and could be merged.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat (hunk 3 above)
Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat]: If you want to make sure that the value you use OK, you can follow https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideSubgroupMC#Measure_proper_value_of_qCut

Comment (efeyazgan) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/exo_diboson/Spin-0/Radion_VBF_WW_inclu/Radion_VBF_WW_inclu_narrow_M/Radion_VBF_WW_inclu_narrow_M_run_card.dat]: Sorry wrong chat.

**Pull request merged**